Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -43,59 +43,83 @@ You can simply select _DefaultMixedRealityToolkitConfigurationProfile_ and then
     
     - For **Platform(s)** select _'Windows Universal'_.
 
-![MRTK](../Images/EyeTracking/mrtk_setup_eyes_dataprovider.png)
+![MRTK](../Images/EyeTracking/mrtk_setup_eyes_dataprovider.jpg)
```

Comment:
General question - why the migration from png to jpg?

Reply:
I had to update the screenshots anyways and did it for consistency and smaller file sizes.